mlir/test/Target/Cpp/global.mlir

```diff
@@ -36,3 +36,6 @@ func.func @use_global(%i: index) -> f32 {
   // CHECK-SAME: (size_t [[V1:.*]])
   // CHECK:   return myglobal[[[V1]]];
 }
+
+emitc.global @ref : i32 = #emitc.opaque<"myglobal_int"> ref
```

Comment:
Is it enforced that globals of ref type are initialized?

Reply:
Now they are. The value must be a `opaque` attribute for now, so we can't analyze it further and make sure it is an lvalue identifier.